files/class-vip-filesystem-stream-wrapper.php

```diff
@@ -114,6 +114,8 @@ class VIP_Filesystem_Stream_Wrapper {
 	 */
 	private $should_flush_empty;
 
+	public static ?API_Client $default_client = null;
```

Comment:
This is to make this class more testable. If we call the FS-related function in the `vip://` inside the class, it will be difficult to override the client without this change.